src/app/snapshot/containers/snapshots-page/snapshots-page.container.ts

```diff
@@ -25,7 +25,7 @@ import { snapshotPageSelectors, vmSnapshotsSelectors } from '../../store';
     ></cs-snapshots-page>`,
 })
 export class SnapshotsPageContainerComponent implements OnInit, AfterViewInit {
-  readonly snapshots$ = this.store.pipe(select(fromSnapshots.selectFilteredSnapshots));
+  readonly snapshots$ = this.store.pipe(select(snapshotPageSelectors.getFilteredSnapshots));
```

Comment:
Why did you rename select -> get? I think we should keep "select", because it's the default prefix in NgRx Entity:
`export const { selectIds, selectEntities, selectAll, selectTotal } = adapter.getSelectors(state);`